Week-3/memoize_assignment.js

```diff
@@ -0,0 +1,33 @@
+// Assignment for creating own memoize function
+// avoiding to use the library 'loadash/memoize' or from underscore
+
+function memoize(fun) {
```

Comment:
@rajeshvlega9 looks good to me. But i could see week2 assignments code as well. Can you merge the previous one